plans/ci.tf

```diff
@@ -0,0 +1,122 @@
+#
+# Resources related to our CI infrastructure for ci.jenkins.io or trusted.ci
+#
+
+
+resource "azurerm_resource_group" "ci" {
+    name     = "${var.prefix}jenkinsci"
+    location = "${var.location}"
+    tags {
+        env = "${var.prefix}"
+    }
+}
+
+resource "azurerm_virtual_network" "ci_vnet" {
+  name                = "ci_vnet"
+  address_space       = ["10.0.0.0/16"]
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.ci.name}"
+}
+
+resource "azurerm_subnet" "ci_subnet" {
+  name                 = "acctsub"
+  resource_group_name  = "${azurerm_resource_group.ci.name}"
+  virtual_network_name = "${azurerm_virtual_network.ci_vnet.name}"
+  address_prefix       = "10.0.2.0/24"
+}
+
+resource "azurerm_storage_account" "ci_storage" {
+  name                     = "${var.prefix}jenkinscistore"
+  resource_group_name      = "${azurerm_resource_group.ci.name}"
+  location                 = "eastus2"
+  account_type             = "Standard_LRS"
```

Comment:
In terraform 0.10, account_type is replaced by     
----
account_tier             = "Standard"
account_replication_type = "LRS"
----